tensorflow/lite/kernels/split.cc

```diff
@@ -151,10 +152,8 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       break;
     }
     default:
-      context->ReportError(context,
-                           "Only float32, uint8, int8, int16 and int32 are "
-                           "currently supported, got %d.",
-                           op_context.input->type);
+      context->ReportError(context, "Type %s currently not supported.",
```

Comment:
I think it's still useful to keep the "Only float32....supported" part.

Reply:
@haozha111 : Yes it is useful and there is no issue in it currently, but it is kind of static message, so modified it, so that later if some new type support added, no need to worry to update the error message.